charts/scalar-manager/templates/deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "scalar-manager.fullname" . }}
+  labels:
+    {{- include "scalar-manager.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "scalar-manager.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "scalar-manager.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "scalar-manager.serviceAccountName" . }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - containerPort: {{ .Values.scalarManager.port }}
+          env:
+            - name: PORT
+              value: "{{ .Values.scalarManager.port }}"
+            - name: NUXT_SCALAR
+              value: {{ .Values.scalarManager.config | toJson | quote }}
```

Comment:
The configuration for Scalar Manager is passed to the container via the environment variable `NUXT_SCALAR`.

About the configuration, see details here
https://github.com/scalar-labs/scalar-manager#configuration